src/ui/HW20/addNewProductFormValidation.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { faker } from "@faker-js/faker";
+import { MENUITEM } from "../../data/types/leftSideMenuItems.types.js";
+import { AddNewProductService } from "../services/products/addNewProduct.service.js";
+import { ProductsListService } from "../services/products/products.service.js";
+import { SalesPortalService } from "../services/salesPortal.service.js";
+import { SignInService } from "../services/signIn.service.js";
+import { generateNewProduct } from "../../data/newProduct.js";
+import { ModalWindowService } from "../services/products/modalWindow.service.js";
+import { IProduct } from "../../data/types/product.types.js";
+import { INPUTFIELD } from "../../data/textNotification.js";
+
+
+describe('Validation form', () => {
+    
+    const signInService = new SignInService();
+    const salesPortalService = new SalesPortalService();
+    const addNewProductService = new AddNewProductService();
+    const productsService = new ProductsListService();
+    const modalWindowService = new ModalWindowService();
+
+    beforeEach(async() => {
+        await signInService.openSalesPortal();
+        await signInService.loginAsAdmin();
+        await salesPortalService.openPageFromLeftsideMenu(MENUITEM.PRODUCTS)
+        await productsService.openAddNewProductPage();
+    });
+
+    afterEach(async() => {
+        await signInService.signOut();
+    });
+
+    context('Positive cases', () => {
+        let newProduct: IProduct
+
+        afterEach(async() => {
+            await productsService.deleteCreatedProduct(newProduct);
+        });
+
+        it('create new product with data minimum values', async () => {
```

Comment:
Пробуй  именовать тестьі, чтоб название говорило само за себя
```suggestion
        it('Should successfully create new product with minimum input values', async () => {
```